docs/about/stay-connected.rst

```diff
@@ -27,16 +28,25 @@ or moving them there if they develop into something bigger.
 
 Learn more about our :doc:`/slack`.
 
-Twitter
-~~~~~~~
+Job Board
+~~~~~~~~~
 
-We're pretty active on `Twitter <https://twitter.com/hashtag/writethedocs?f=tweets&vertical=default&lang=en>`__ as well, mostly on **#writethedocs**.
+Write the Docs now has a :doc:`/job-board`.
+This is the place to go if you want to hire or be hired by companies in the Write the Docs community.
```

Comment:
I'd actually remove 

> in the Write the Docs community

seems limiting for no need. And if companies don't feel part of the community they might not approach